src/App.js

```diff
@@ -1,25 +1,20 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import BitByteNode from './BitByteNode.jsx'
 
 function App() {
+
+  /* Declaring as a variable for function call */
```

Comment:
You said you wanted to use addBit() to add extra nodes to the BitByteNode class? Why not just declare the function here and pass it down with the state as a prop?

Reply:
Sounds like a good idea. However, I no longer find the need to add such a function. I can do what matches Shravan's model, which basically connects two nodes by id.
Shall I create a Edge component like Shravan did in his code?

Reply:
Seems good. Would this component basically draw a line between the parent note and multiple child nodes?

Reply:
That's the goal. I can use id as a key to retrieve the nodes and connect them.
The current concern is how I would visually connect the nodes? I came across the solution of rendering the nodes as svg components, but don't think it would be ideal for this project

Reply:
If you're interested in using a library this one has already been made https://www.npmjs.com/package/react-d3-tree

Reply:
Sounds good, I pushed some changes with hard-coded data
The major change will be in src/FamilyTree.jsx